contracts/interfaces/IIdleMStableStrategy.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+
+import "./IIdleCDOStrategy.sol";
+
+interface IIdleMStableStrategy is IIdleCDOStrategy {
+    function transferShares(
+        address _to,
+        uint256 _interestTokens,
+        uint256 _govShares
+    ) external;
+}
```

Comment:
This interface seems to be unused in `mstable/IdleMstableStrategy.sol`

Reply:
removed the unused interface